brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3263 Fix: Global - Resolved a significant issue of multiple `_uag_page_assets` entries in the database when using an FSE theme.

### Description

- Resolve 2nd Issue reported on FB - https://www.facebook.com/groups/wpspectra/permalink/829864115101663/?mibextid=Nif5oz
- Slack link - https://brainstormforce.slack.com/archives/C03TMLQG0EN/p1692237131276689
- Task link - https://brainstormforce.atlassian.net/browse/SPCTR-3263

### Screenshots

- https://www.awesomescreenshot.com/video/20007597?key=af919d95bc97cb83b747178e90e8eaa4

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Switch to Master and use any FSE theme.
- Create an Page with Spectra block on it. ( Style the block with color )
- Visit the frontend.
- Go to database and select `wp_postmeta`.
- You will see whenever you refresh the page entries of  `_uag_page_assets` get increase in DB
- Now switch to the branch and refresh the page again. make sure DB entry of `_uag_page_assets` does not increase.
- Styling should be proper on frontend. Edit style in backend, update and refresh the page. Make sure only `_uag_page_assets` should be same only styling inside it get updated with change in `meta_id`.

### Checklist:
